docs/language.rst

```diff
@@ -125,7 +125,17 @@ Literal values can be assigned to variables
 Primitive types
 ==============================
 
-The basic primitive types are ``string``, ``number``, ``int`` or ``bool``.
+The basic primitive types are ``string``, ``number``, ``int`` or ``bool``. These basic types also support type casts:
+
+.. code-block:: inmanta
+
+    assert = true
```

Comment:
Is this too meta for the docs? What is most readable?
- assignments without assertion: `x = int("1")`?
- type cast expressions without assignment: `int("1")?
- something else?

Reply:
It is probably a bit hard to get the meaning of this assert statement here for newbies, but I think it doesn't degrade the utility of the code snippet, which is mainly (I think) showing of the syntax.  (I actually never did it like that in the past, I like it a lot)
I would keep it like that